sdk/spring-sdk/src/main/scala/kalix/springsdk/impl/ActionDescriptorFactory.scala

```diff
@@ -73,10 +77,30 @@ private[impl] object ActionDescriptorFactory extends ComponentDescriptorFactory
           .withKalixOptions(kalixOptions)
       }
 
+    val subscriptionEventSourcedEntityClass =
+      if (hasEventSourcedEntitySubscription(component)) {
+        component.getMethods.sorted // make sure we get the methods in deterministic order
+          .filter(hasActionOutput)
+          .collect {
+            case method
+                if !subscriptionEventSourcedEntityMethods.exists(s =>
+                  s.serviceMethod.methodName == method.getName) => // individual annotated overrides class level annotation
```

Comment:
also not needed

Reply:
I specifically looked for it... and failed